src/main/java/pl/wedding/management/model/employee/StatusEmployee.java

```diff
@@ -0,0 +1,5 @@
+package pl.wedding.management.model.employee;
+
+public enum StatusEmployee {
+    WORKING, VACATION, SICKNESS, FIRED
```

Comment:
Super pomysł!

Reply:
Pomysł zgadza się ekstra, ja bym tylko od siebie dodał, żeby ujednolicić to w stylu: albo przymiotniki albo rzeczowniki. Jak przymiotniki to coś w stylu: [working, off, sick, fired]

Reply:
podpisuję się rękoma i nogami. :)